modules/deploy/pages/deployment-option/cloud/managed-connectors/sizing-connectors.adoc

```diff
@@ -0,0 +1,26 @@
+= Sizing Connectors
+:description: How to choose number of tasks to set for a connector.
+:page-aliases: cloud:managed-connectors/task-count.adoc
+:page-cloud: true
+
+== What is a connector task?
+When you set up a connector, its main responsibility is to validate the provided configuration and spawn tasks.
+The actual work is performed by the connector tasks. Setting multiple tasks for a connector allows to pararelize the
```

Comment:
```suggestion
The actual work is performed by the connector tasks. Setting up multiple tasks for a connector allows for parallelization of the
```

Reply:
We have a new glossary and should probably add a definition for _connector task_.
Can you provide one? 
This heading says the section serves to explain connector tasks, but after reading it, I'm still unclear about what, exactly, a connector task is beyond it being the mechanism that "does the work" <---what do you mean by this? thx.

Reply:
A great way to start this explanation: 
For example, connector tasks <perform task a, task b, and task c>...

Reply:
`A task is a thread that performs the actual copying of data.`
Where to put the definition?